AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.java

```diff
@@ -217,4 +226,51 @@ public FunctionItem createFromParcel(Parcel in) {
             void consume(AnkiActivity activity);
         }
     }
+
+    private static class ExceptionReportItem extends Item implements Parcelable{
+
+        UUID uuid = UUID.randomUUID();
+        private static final Map<String, Long> lastClickMap = new HashMap<>();
+        final int minInterval = 60000;
+        final String lastClick = "LAST_CLICK";
+
+        public ExceptionReportItem(@StringRes int titleRes, @DrawableRes int iconRes) { super(titleRes, iconRes); }
+
+        @Override
+        public void execute(AnkiActivity activity) {
+            long currentTimestamp = SystemClock.uptimeMillis();
+            Long lastClickTimeStamp = lastClickMap.get(lastClick);
+
+            if (lastClickTimeStamp == null || currentTimestamp - lastClickTimeStamp > minInterval) {
+                AnkiDroidApp.sendExceptionReport(
+                        new Throwable("Exception report sent by user manually"),
+                        "AnkiDroidApp.HelpDialog",
+                        uuid.toString());
+                Toast.makeText(activity, "Report Submitted.", Toast.LENGTH_SHORT).show();
```

Comment:
Toasts shouldn't end in a full stop

```suggestion
                Toast.makeText(activity, "Report submitted", Toast.LENGTH_SHORT).show();
```